### ISSUE TITLE
Support for neocomplete

### ISSUE THREAD

**Issue by pantierra** [veloce/vim-behat#1]
Behat doesn't seem to work with neocomplete, only with it's predecessor neocomplcache. Wouldn't it make sense to support it?

**Comment by veloce** [comment]
Yep it would definitely make sense. Unfortunately I don't use Behat anymore, and I don't maintain the plugin anymore... 
Feel free to adapt it to neocomplete, it should not be hard as it was only configuration.
